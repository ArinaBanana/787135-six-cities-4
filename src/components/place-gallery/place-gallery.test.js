import React from "react";
import renderer from "react-test-renderer";
import PlaceGallery from "./place-gallery.jsx";

const images = [`path`, `path`];

it(`Should render Place Gallery`, () => {
  const tree = renderer.create(<PlaceGallery images={images} />).toJSON();

  expect(tree).toMatchSnapshot();
});
