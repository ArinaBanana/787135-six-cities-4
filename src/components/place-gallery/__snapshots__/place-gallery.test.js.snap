// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Place Gallery 1`] = `
<div
  className="property__gallery"
>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="path"
    />
  </div>
  <div
    className="property__image-wrapper"
  >
    <img
      alt="Photo studio"
      className="property__image"
      src="path"
    />
  </div>
</div>
`;
